share/picongpu/tests/Pusher/etc/picongpu/1.cfg

```diff
@@ -0,0 +1,78 @@
+# Copyright 2013-2022 Heiko Burau, Rene Widera, Felix Schmitt, Axel Huebl,
+#                     Franz Poeschel
+#
+# This file is part of PIConGPU.
+#
+# PIConGPU is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# PIConGPU is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with PIConGPU.
+# If not, see <http://www.gnu.org/licenses/>.
+#
+
+##
+## This configuration file is used by PIConGPU's TBG tool to create a
+## batch script for PIConGPU runs. For a detailed description of PIConGPU
+## configuration files including all available variables, see
+##
+##                      docs/TBG_macros.cfg
+##
+
+
+#################################
+## Section: Required Variables ##
+#################################
+
+TBG_wallTime="1:00:00"
+
+TBG_devices_x=1
+TBG_devices_y=1
+TBG_devices_z=1
+
+TBG_gridSize="64 64 32"
+TBG_steps="100"
+
+TBG_periodic="--periodic 1 1 1"
+
+
+#################################
+## Section: Optional Variables ##
+#################################
+
+# write position to stdout (messy):
+# --e_position.period 1
+
+TBG_openPMD="--openPMD.period 1 --openPMD.file simData --openPMD.ext bp"
+TBG_plugins="!TBG_openPMD \
+             --e_macroParticlesCount.period 100"
```

Comment:
Do you use the macro particle counter plugin? If not, t could be removed.

Reply:
done